hostinit/setup.go

```diff
@@ -244,6 +244,7 @@ func (init *HostInit) setupReadyHosts(ctx context.Context) error {
 						catcher.Add(errors.New("hostinit run canceled"))
 						return
 					}
+					time.Sleep(5 * time.Second)
```

Comment:
We noted that reachability check and SCP were happening within the same second, and think trying this would yield some useful info.

Reply:
can we ramp up to 5 seconds? try it with 1 or 2 and see if that's enough? 

another option, that I think might be worth trying is running the echo hi step in a loop inside of the ishostready that, if it succeeds, it sleeps for a number of seconds and makes sure it succeeds a second time? Also maybe recording and optionally logging some of the output of the ssh command (with increased verbosity?) so we have a better idea of how things work. 

I'm fine deploying this. Less fine merging it in.

Reply:
I did a different thing. PTAL.